cmd/test-resources/repos/test-repo/ttpforge-repo-config.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
bringing file-based E2E tests back now that the codebase has stabilized to ensure that bugs like this are covered in the future